app/views/layouts/_header.html.erb

```diff
@@ -1,12 +1,12 @@
 <% case try(:current_namespace) %>
 <% when 'candidate_interface' %>
-  <%= render(HeaderComponent.new(classes: "app-header--#{HostingEnvironment.environment_name}",
+  <%= render(HeaderComponent.new(classes: "app-header--#{HostingEnvironment.environment_name} govuk-!-display-none-print",
```

Comment:
I wonder if we should just add this to the `HeaderComponent`? It's now `govuk-!-display-none-print` everywhere it's used

Reply:
I did try to add it directly to the header component template, but it wasn't working. Was hoping that someone could help me understand why, seeing as it worked here when injected as the arg

Reply:
Sorted it out, turns out there's more than one type of header component, so I've flagged the correct ones.